src/App.js

```diff
@@ -1,50 +1,56 @@
-import './App.css';
+import './style/App.css';
 import { useEffect, useState } from "react";
-import { ShowPokemon } from './showPokemon';
+import { ShowPokemon } from './components//showPokemon';
+import { Button } from './components/buttons';
```

Comment:
Is it `Button` or `Buttons`? File name usually matches the component name. What do those buttons do? Sounds like Pagination